index.rst

```diff
@@ -24,16 +24,13 @@ that is demonstrated by the notebook, a description of the data that are used,
 and a listing of the data analysis tools that
 are demonstrated by the use case.
 
-The latest versions of the notebooks are contained in the
-`GitHub jdat_notebooks repository <https://github.com/spacetelescope/jdat_notebooks>`_.
-
-
 Installation Instructions
 =========================
 
 To download and execute the notebooks, we recommend that you
-clone this repository to your local computer as described
-in :ref:`installation instructions <install>` for details.
+clone the `jdat_notebooks repository <https://github.com/spacetelescope/jdat_notebooks>`_
+to your local computer as described
+in the :ref:`installation instructions <install>`.
```

Comment:
When viewing the page, all of the text, including ":ref: and "<install>" is visible.

Reply:
I believe this is normal and the correct display happens when the docs are built. See e.g. this https://github.com/spacetelescope/jdaviz/blob/main/docs/index.rst

Reply:
Ah, I see. All good then.